src/v/archival/ntp_archiver_service.cc

```diff
@@ -86,23 +86,29 @@ const cloud_storage::manifest& ntp_archiver::get_remote_manifest() const {
 ss::future<cloud_storage::download_result>
 ntp_archiver::download_manifest(retry_chain_node& parent) {
     gate_guard guard{_gate};
-    retry_chain_node fib(manifest_upload_timeout, 100ms, &parent);
```

Comment:
nit: I would separate out the changes to improve logging with the other changes to add listobjectsv2 support & explicit key support

Reply:
rearranged the commits, all logging changes are in one commit now